bigbluebutton-config/bin/bbb-record

```diff
@@ -48,10 +48,10 @@
 source /etc/bigbluebutton/bigbluebutton-release
 
 BBB_USER=bigbluebutton
-BBB_PROPERTIES=/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
 BBB_YML=/usr/local/bigbluebutton/core/scripts/bigbluebutton.yml
 
-SERVER_URL=$(sed -n '/^bigbluebutton.web.serverURL/{s/.*\///;p}' < $BBB_PROPERTIES)
+# BBB_PROPERTIES=/usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties
```

Comment:
Please don't leave commented code hanging around, at least without a comment explaining why someone might want to uncomment and use this code on a live BBB system.

Reply:
Should I just remove it, then? This variable was previously declared but not used anywhere.

Reply:
Yeah, if the code is unused and you can't think of a reason someone would uncomment it on a running system, then just remove it.